ci-operator/config/openstack-k8s-operators/openstack-operator/openstack-k8s-operators-openstack-operator-master.yaml

```diff
@@ -1,44 +1,40 @@
 base_images:
+  cli:
+    name: "4.10"
+    namespace: ocp
+    tag: cli
   golangci-lint:
     name: golangci-lint
     namespace: ci
     tag: v1.46.0
-  openstack-operator-ci-pre-commit:
-    name: tools
-    namespace: openstack-k8s-operators
-    tag: openstack-operator-ci-pre-commit
-binary_build_commands: make build
+  opm-builder:
+    name: opm-builder
+    namespace: ci
+    tag: latest
 build_root:
-  image_stream_tag:
-    name: release
-    namespace: openshift
-    tag: rhel-8-release-golang-1.18-openshift-4.12
+  from_repository: true
```

Comment:
It will get build_root from:
https://github.com/openstack-k8s-operators/openstack-operator/blob/master/.ci-operator.yaml